applications/FluidDynamicsApplication/custom_constitutive/newtonian_2d_law.cpp

```diff
@@ -18,6 +18,11 @@
 // Project includes
 #include "includes/cfd_variables.h"
 #include "includes/checks.h"
+
+// Application includes
+#include "custom_constitutive/newtonian_law/newtonian_2d_adjoint_law.h"
```

Comment:
Newtonian CLaw was conceived to be add simple as possible. I think we are adding too much complexity.

Correct me if I'm wrong, your design aims at avoiding the user to set the adjoint constitutive law. Hence, the user selects the newtonian one, which internally points to its adjoint.

Reply:
That is correct. There are few reasons for it.
1. The interface introduced here for the adjoint CLs are not available in the base CL, hence either I have to put it in the base CL which is too complicated (Base CL is too complicated to even make a adjoint CL.). So I thought to create an adjoint CL. This CL is created without inheritance for easy decoupling.
2. If I want to enable user to select the correct adjoint CL, then I need to register this Adjoint base class to the component factory which I thought at this point not relevant.

Reply:
Now using the base class interface, hence this is removed.